apps/avatax/e2e/data/maps/channel.json

```
@@ -2,7 +2,7 @@
   "Channel": {
     "USA": {
       "id": "Q2hhbm5lbDoz",
-      "slug": "channel-us",
+      "slug": "default-channel",
```

issue: We cannot change the `slug`, since it won't work on our testing environment: https://shopex-avatax-318.eu.saleor.cloud/dashboard/channels/Q2hhbm5lbDoz

I know that for local testing this setup is a bit problematic, maybe we could create a separate environment in Saleor Cloud and use the same snapshot data we use for testing environment 😅

```suggestion
      "slug": "channel-us",
```

---

oh, you're right, I forgot to change that 🤦 thanks 😅